index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Microverse Player</title>
+	<link rel="stylesheet" type="text/css" href="assets/style.css" />
+</head>
+
+<body>
+	<header>
+		<nav>
+			<ul class="nav">
+				<li class="menu_bar">
+					<a href="">
+							<img src="https://img.icons8.com/metro/26/000000/line-width.png">
+					</a>
+				</li>
+				<li class="nav__logo">
+					<a href="www.youtube.com"><img class="logo"
+							src="https://i.kinja-img.com/gawker-media/image/upload/s--ohRBA8uj--/c_scale,f_auto,fl_progressive,q_80,w_800/pol5zfmalpqmpt1jcnbp.png"
+							alt="youtube image" /></a>
+				</li>
+				<!--
+				-->
+				<li class="nav__searchbar">
+					<input placeholder="search for a video" />
+				</li>
+				<!-- 		
+
+	  		-->
+				<li class="nav__searchbar-button">
+					<a><input type="button" title="Search" /></a>
+				</li>
+				<li>
+					<a href="">
+						<img class="icons"
+							src="https://img.icons8.com/ios/50/000000/video-call.png" />
+					</a>
+				</li>
+				<li>
+					<a href="">
+						<img class="icons"
+							src="https://img.icons8.com/material/50/000000/squared-menu--v1.png"
+							alt="" /></a>
+				</li>
+				<li>
+					<a href="">
+						<img class="icons"
+							src="https://img.icons8.com/ios-glyphs/30/000000/moved-topic.png"
+							alt="" /></a>
+				</li>
+				<li>
+					<a href=""><img class="icons"
+							src="https://img.icons8.com/ios-glyphs/30/000000/appointment-reminders.png"
+							alt="" /></a>
+				</li>
+				<li>
+					<a href="">
+						<img src="https://randomuser.me/api/portraits/women/85.jpg"
+							class="icons avatar">
+					</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<hr>
+	<main>
+		<div class="main">
+			<section class="col-2-3">
+				<iframe width="100%" height="415" src="https://www.youtube.com/embed/jOyfZex7B3E"
```

Comment:
width in % is not allowed for iframe, use numbers instead .

Reply:
done :+1:

Reply:
Fixed, we used pixels instead.